src/terminal/adapter/adaptDispatch.cpp

```diff
@@ -3517,12 +3447,6 @@ bool AdaptDispatch::SetCursorColor(const COLORREF cursorColor)
 // - True if handled successfully. False otherwise.
 bool AdaptDispatch::SetClipboard(const wil::zwstring_view content)
 {
-    // Return false to forward the operation to the hosting terminal,
-    // since ConPTY can't handle this itself.
-    if (_api.IsConsolePty())
-    {
-        return false;
-    }
     _api.CopyToClipboard(content);
```

Comment:
Won't this result in conhost *also* trying to write to the clipboard when it gets OSC52?

This was guarded to avoid the conpty server overwriting the clipboard or fighting with the conpty client.

Reply:
`ConhostInternalGetSet::CopyToClipboard` is currently not implemented. I think, if anything, the decision whether it should copy to the clipboard should happen in `ConhostInternalGetSet` depending on whether a proper HWND exists or not.